decidim-resource_banks/lib/decidim/resource_banks/participatory_space.rb

```diff
@@ -51,6 +51,7 @@
         banner_image: File.new(File.join(seeds_root, "city2.jpeg")), # Keep after organization
         published_at: 2.weeks.ago,
         authorship: Faker::Name.name,
+        scope: n.positive? ? nil : Decidim::Scope.reorder(Arel.sql("RANDOM()")).first,
```

Comment:
Currently the var **n** does not exist in this block we should rename **_n** to **n** in the declaration above. https://github.com/gencat/participa-catalunya/blob/53cec2f1148983281181ebce7754a9bb7ce4d02b/decidim-resource_banks/lib/decidim/resource_banks/participatory_space.rb#L39